test/serveable_file_test.rb

```diff
@@ -467,6 +467,12 @@ def test_has_static_extension_returns_true_for_flash
                "Should handle flash .swf files"
       end
 
+      def test_has_static_extension_returns_true_for_configured_extension
+        Rack::Zippy.static_extensions << 'csv'
```

Comment:
What do you think about resetting the static extensions to their original value in the teardown?